conans/test/integration/editable/test_editable_python_requires.py

```diff
@@ -0,0 +1,41 @@
+import os
+import re
+import textwrap
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient
+
+
+def test_editable_python_requires():
+    c = TestClient()
+    dep = textwrap.dedent("""
+        from conan import ConanFile
+
+        number = 42
+
+        class Dep(ConanFile):
+            name = "dep"
+            version = "1.0"
+        """)
+    pkg = textwrap.dedent("""
+        from conan import ConanFile
+
+        number = 42
```

Comment:
Whi is this number = 42 needed here for this test?

Reply:
sorry, this test slipped here, not belongs here, I will remove it.